backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -665,14 +676,64 @@ public static List<FeedbackMessage> validatePositiveHIVRequiredAOEFields(
               .scope(ITEM_SCOPE)
               .fieldHeader(pregnant.getHeader())
               .source(ResultUploadErrorSource.SIMPLE_REPORT)
-              .message(getRequiredHivAoeValuesErrorMessage(pregnant.getHeader()))
+              .message(
+                  getPositiveResultRequiredValueErrorMessage(
+                      pregnant.getHeader(), DiseaseService.HIV_NAME))
               .errorType(ResultUploadErrorType.MISSING_DATA)
               .fieldRequired(pregnant.isRequired())
               .build());
     }
     return errors;
   }
 
+  private static FeedbackMessage buildSyphilisMissingDataFeedbackMessage(ValueOrError field) {
+    return FeedbackMessage.builder()
+        .scope(ITEM_SCOPE)
+        .fieldHeader(field.getHeader())
+        .source(ResultUploadErrorSource.SIMPLE_REPORT)
+        .message(
+            getPositiveResultRequiredValueErrorMessage(
+                field.getHeader(), DiseaseService.SYPHILIS_NAME))
+        .errorType(ResultUploadErrorType.MISSING_DATA)
+        .fieldRequired(field.isRequired())
+        .build();
+  }
+
+  public static List<FeedbackMessage> validatePositiveSyphilisRequiredAOEFields(
+      ValueOrError testResult,
+      ValueOrError gendersOfSexualPartners,
+      ValueOrError pregnant,
+      ValueOrError previousSyphilisDiagnosis,
+      ValueOrError symptomaticForDisease) {
+    List<FeedbackMessage> errors = new ArrayList<>();
+    // includes SNOMED values for positive and detected
+    Set<String> positiveTestResultValues =
+        Set.of(POSITIVE_LITERAL, DETECTED_LITERAL, POSITIVE_CODE, DETECTED_CODE);
+    if (!positiveTestResultValues.contains(testResult.getValue().toLowerCase())) {
```

Comment:
looking at [this constructor for `ValueOrError`](https://github.com/CDCgov/prime-simplereport/blob/db562d21c605a529b6d7b421bc77ad75803e70f3/backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java#L743) would we want to do a null check for cases where `testResult` is malformed / null like we do for the AOE questions? In those cases think we might get an NPE here trying to cast it to lower case